Resources/views/Collector/mongodb.html.twig

```diff
@@ -34,21 +34,43 @@
 {% endblock %}
 
 {% block panel %}
-    <h2>Commands</h2>
+    <h2>Command metrics</h2>
+    <div class="metrics">
+        <div class="metric">
+            <span class="value">{{ collector.commandCount }}</span>
+            <span class="label">Database Queries</span>
+        </div>
+
+        <div class="metric">
+            <span class="value">{{ '%0.3f'|format(collector.time / 1000) }} ms</span>
+            <span class="label">Query time</span>
+        </div>
+    </div>
 
+    <h2>Commands</h2>
     {% if collector.commands is empty %}
-        <p>
-            <em>No commands.</em>
-        </p>
-    {% else %}
-        <ul class="alt">
-            {% for command in collector.commands %}
-                <li class="{{ cycle(['odd', 'even'], loop.index) }}">
-                    <div>
-                        <code>{{ command }}</code>
-                    </div>
-                </li>
-            {% endfor %}
-        </ul>
+        <div class="empty">
+            <em>No commands were performed.</em>
+        </div>
     {% endif %}
+    <table>
+        <thead>
+        <tr>
+            <th class="nowrap">#</th>
+            <th class="nowrap">Time</th>
+            <th class="nowrap">Database</th>
+            <th style="width: 100%;">Info</th>
+        </tr>
+        </thead>
+        <tbody id="commands">
+        {% for command in collector.commands %}
+            <tr>
+                <td class="nowrap">{{ loop.index }}</td>
+                <td class="nowrap">{{ '%0.3f'|format(command.durationMicros / 1000) }} ms</td>
+                <td class="nowrap">{{ command.database }}</td>
+                <td>{{ command.command }}</td>
```

Comment:
Raw JSONs are hard to read so I think we could add a collapsible pretty-print version of the command JSON here just like in ElasticaBundle: https://github.com/FriendsOfSymfony/FOSElasticaBundle/blob/master/src/Resources/views/Collector/elastica.html.twig#L129-L135

Reply:
What about decoding and encoding current value in template to keep canonical extended JSON format ? Like `{{ data|json_decode|json_encode(constant('JSON_PRETTY_PRINT')) }}`.

Reply:
Yes, that makes sense 👍🏻

Reply:
Done. FYI, `json_decode` has been done in DataCollector since Twig does not provide a such filter, cf. https://github.com/twigphp/Twig/pull/824.